templates/apollo-starter-ts/src/resolvers/generated/test.ts

```diff
@@ -0,0 +1,25 @@
+import { GraphQLContext } from '../../context'
+
+export const testResolvers = {
```

Comment:
Did you mean for this file to be committed?

Reply:
ups..